cartoframes/data/observatory/repository/category_repo.py

```diff
@@ -0,0 +1,41 @@
+from cartoframes.data.observatory.repository.repo_client import RepoClient
+
+
+def get_category_repo():
+    return CategoryRepository()
+
+
+class CategoryRepository(object):
+
+    def __init__(self):
+        self.client = RepoClient()
+
+    def get_all(self):
+        return self._to_categories(self.client.get_categories())
+
+    def get_by_id(self, category_id):
+        result = self.client.get_categories('id', category_id)
+
+        if len(result) == 0:
+            return None
```

Comment:
I usually prefer raising errors instead of forcing the consumer of this method to check for `None`. Also it's easier to document that a method raises errors than explaining in which cases it returns None.

This comment applies to the rest of repositories.

Reply:
```python
catalog.categories # [{'id': 'finacial'}, {'id': 'housing'}, ...]
catalog.categories.get_by_id('non-existent') # error or None
```

`list` can return an exception and None. In a DataFrame, if you try to get a unexisting column on index, it will return an error. So, I would go with the error

Reply:
OK, I'll change it in the next PR :)